sklearn/decomposition/_kernel_pca.py

```diff
@@ -346,6 +346,19 @@ def transform(self, X):
     def inverse_transform(self, X):
         """Transform X back to original space.
 
+        ``inverse_transform`` approximates the inverse transformation using
+        a learned pre-image. The pre-image is learned by kernel ridge
+        regression of the original data on their low-dimensional representation
+        vectors.
```

Comment:
I think that I would add a note mentioning about the mean. Such a proposal (feel free to modify)

```
.. note::
   :meth:`~sklearn.decomposition.fit` will internally used a centered
   kernel. This information is then loss and `inverse_transform`
   cannot take this original centering into account.
```